### ISSUE TITLE
How to create many time to loop differ many time martingale ? 

### ISSUE THREAD

**Issue by mrburnass** [binary-com/binary-bot#1527]
How do I have many time to loop?
if stake 11 loss -> 22 martingale 6 time back to 11
if stake 44 loss -> 88 martingale 6 time back to 11
if stake 88 loss -> 176 martingale 6 time back to 11
if stake 176 loss -> 352 martingale 6 time back to 11
and next condition...
stake every stake
loss = ((stake loss)*10/5) loop 6 time

Please advise me about loop
https://drive.google.com/open?id=1hxccyIkw7qAzoMxp39qHx__7AretyvQp
[binary-differ.zip](https://github.com/binary-com/binary-bot/files/3194729/binary-differ.zip)

**Comment by romeo1911** [comment]
((stake loss)*10/5)*5 time
with this formula, if u stake 11 ->22  22->44 etc

**Comment by mrburnass** [comment]
> ((stake loss)*10/5)*5 time
> with this formula, if u stake 11 ->22 22->44 etc

yes yes sorry i think u  formula it's well.

**Comment by mrburnass** [comment]
> How do I have many time to loop?
> if stake 11 loss -> 22 martingale 6 time back to 11
> if stake 44 loss -> 88 martingale 6 time back to 11
> if stake 88 loss ->  176 martingale 6 time back to 11
> if stake 176 loss -> 352 martingale 6 time back to 11
> and next condition...
> stake every stake
> loss = ((stake loss)*10/5)*5 time
> 
> Please advise me about loop
> https://drive.google.com/open?id=1hxccyIkw7qAzoMxp39qHx__7AretyvQp
> [binary-differ.zip](https://github.com/binary-com/binary-bot/files/3194729/binary-differ.zip)

**Comment by mrburnass** [comment]
![1](https://user-images.githubusercontent.com/31042419/57997144-4b468600-7af5-11e9-8be2-0b61cc4148d3.png)
![2](https://user-images.githubusercontent.com/31042419/57997152-51d4fd80-7af5-11e9-9ba6-87ab8b29a354.png)

**Comment by Hooponopono** [comment]
nice nickname, u eat too much chili?
btw your bot is like this:
if result win stake =1
if result is loss stake *1.095
if result is loss stake to 1
if result is win stake *2*6
your method,  a lot of people try and is not working, with loop of 6, 2 or 10 or whatever
